earthly/postgresql/scripts/std_docs.py

```diff
@@ -0,0 +1,438 @@
+            + f"{excludes}"
+            + f"{comments}"
+            + f"{column_description_wrap}"
+            + f"{table_description_wrap}"
+            + f" | dot -Tsvg -o {filename}",
+            # + f" > {filename}.dot",
+            name=f"Generate Schema Diagram: {name}",
+            verbose=True,
+        )
+
+        # if res.ok:
+        #    cli.run(
+        #        f"dot -Tsvg {filename}.dot -o {filename}",
+        #        name=f"Render Schema Diagram to SVG: {name}",
+        #        verbose=True,
+        #    )
+
+        return res
+
+    def full_schema_diagram(self) -> cli.Result:
+        # Create a full Schema Diagram.
+        return self.dbviz(
+            "docs/full-schema.svg",
+            "Full Schema",
+            self.schema_name(),
+            excluded_tables=self.full_schema_excluded_tables(),
+            comments=self.full_schema_comments(),
+            column_description_wrap=self.full_schema_column_description_wrap(),
+            table_description_wrap=self.full_schema_table_description_wrap(),
+        )
+
+    def migration_schema_diagram(self, ver: int) -> cli.Result:
+        # Create a schema diagram for an individual migration.
+        if ver in self.migrations:
+            migration = self.migrations[ver]
+
+            include_tables = migration.include(
+                self.all_schema_included_tables(), self.all_schema_excluded_tables()
+            )
+            if include_tables is None:
+                return cli.Result(
+                    0,
+                    "",
+                    "",
+                    0.0,
+                    f"Migration {ver} has no tables to diagram.",
+                )
+
+            exclude_tables = migration.exclude(self.all_schema_excluded_tables())
+
+            title = f"{migration.migration_name}"
+            if migration.title and len(migration.title) > 0:
+                title = migration.title
+                
+            comments=None
+            if migration.comments is not None:
+                comments = migration.comments
+            else:
+                comments = self.all_schema_comments()
+
+            return self.dbviz(
+                f"docs/migration-{ver}.svg",
+                f"V{ver}__{migration.migration_name}",
+                title,
+                included_tables=include_tables,
+                excluded_tables=exclude_tables,
+                comments=comments,
+                column_description_wrap=migration.column_description_wrap,
+                table_description_wrap=migration.table_description_wrap,
+            )
+
+    def create_diagrams(self, results: cli.Results) -> cli.Results:
+        # Create a full Schema Diagram first.
+        res = self.full_schema_diagram()
+        results.add(res)
+
+        for ver in sorted(self.migrations.keys()):
+            res = self.migration_schema_diagram(ver)
+            results.add(res)
+
+        # cli.run("ls -al docs", verbose=True)
```

Comment:
Remove this

Reply:
normally I would remove things like this, but this is not production code, its a build time utility.
Its highly possible for it to break because its interogating other files and deriving things from them.
Leaving these debug lines in commented out makes it easier/faster to fix if/when something breaks. 
Its also doing no harm.
So for those reasons, I am inclined to leave them be.